### ISSUE TITLE
Fork this

### ISSUE THREAD

**Issue by mcandre** [cryptaliagy/tomlq#4]
The project has been sitting untouched for some while now.

Could use a fresh fork, and brushing away any CVE's that may have silently crept in from std or crate dependencies over the years.

**Comment by cryptaliagy** [comment]
@mcandre I'm starting to maintain a fork of this now at https://github.com/cryptaliagy/tq-rs

**Comment by jamesmunns** [comment]
Hey @cryptaliagy, are you interested in taking over this repo/the crate on crates-io?

**Comment by cryptaliagy** [comment]
I would be, yes! @jamesmunns

**Comment by cryptaliagy** [comment]
I'll PR in the changes from my fork, and after that I can delete it

**Comment by jamesmunns** [comment]
SGTM!

**Comment by cryptaliagy** [comment]
@jamesmunns I've deleted the repository!

**Comment by jamesmunns** [comment]
@cryptaliagy sent you a request to transfer!

**Comment by jamesmunns** [comment]
@cryptaliagy also sent you an invite for ownership of the `tomlq` crate on crates-io

**Comment by cryptaliagy** [comment]
I've received and accepted the crates-io request, trying to find where the repository transfer requests are

**Comment by jamesmunns** [comment]
https://stackoverflow.com/questions/63244053/how-to-accept-private-repository-ownership-on-github

maybe check your primary email on GH is valid?

**Comment by cryptaliagy** [comment]
Ah thank you, I was checking the wrong email! The repository is now transferred.

I might end up creating a `tomlq` organization account and transferring this repository to it, in case I ever become inactive this should make the process for transferring ownership or adding in new folks in the future a little more streamlined. Are you okay if I keep you as a collaborator in this repository? @jamesmunns

**Comment by cryptaliagy** [comment]
Fair play! I will re-add you to the license, but otherwise I have removed your access as requested

**Comment by cryptaliagy** [comment]
Final updates under this issue:

* `tomlq` is now publishing to the `tomlq` crate in crates.io: https://crates.io/crates/tomlq
* `cargo-binstall` support has been added by archive, and both `tomlq` and `tq` binaries are currently being published. In the new year I will remove the `tomlq` binary and publish version `0.2.0` to finish the transition into the shorter binary name.
* I added a new repository for `tq-rs` in my profile to make sure that existing `tq-rs` users are not impacted, and I've added additional notice on the repository for this migration.
* Dependabot is enabled, and pipeline publishing is successful. Weekly security audit will be performed by `cargo-audit`, and I will look into making sure the binaries are signed down the line.
* Currently I'll leave the repository under my profile, but I will look into migrating it into its own org down the line. I am also potentially interested in additional contributors

For all intents and purposes, I think this wraps up the work for re-integrating the fork into `tomlq` and concludes the requirements of this issue!
